### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by amorabryan**
Regular Browser Recording Issue #1
Private Browser Recording Issue #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,14 +14,62 @@
...
+      <form id="codeJournal">
+        <div class="row">
+          <div class="column-full">
+            <h1>New Entry</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-full column-half">
+            <div class="img">
+              <img src="images/placeholder-image-square.jpg" class="url">
```
Comment (shawnkost) [inline index.html]: Needs an `alt` attribute!

File: js/data.js [hunk 2]
```diff
@@ -6,3 +6,15 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+window.addEventListener('beforeunload', function (event) {
+  event.preventDefault();
```
Comment (shawnkost) [inline js/data.js]: Not needed. We don't need to prevent any action here. We mostly only use this with the `submit` event

File: js/data.js [hunk 3]
```diff
@@ -6,3 +6,15 @@ var data = {
...
+  const journalEntriesJSON = JSON.stringify(data.entries);
```
Comment (shawnkost) [inline js/data.js]: We want to `stringify` the whole `data` object, not just `data.entries`

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  journalData.entryId = data.nextEntryId;
+
+  data.nextEntryId++;
+
+  journalData[title] = titleValue;
+  journalData[photoURL] = photoURLValue;
+  journalData[notes] = notesValue;
+
+  const entriesArray = data.entries;
+  entriesArray.push(journalData);
```
Comment (shawnkost) [inline js/main.js]: `journalData` should be getting added to the beginning of the array

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good so far Bryan, just a few changes requested here! A couple more below that didn't leave comments on.

1. The background color for the navbar should span the whole width of the page. It should never stop.
2. The background color for the main content of the page should also span the whole width of the page. There shouldn't be a gap where you see that white background as well

**Comment by amorabryan** [comment]
Revised Background Color Styling Issue #1

**Comment by amorabryan** [comment]
Revised Background Color Styling Issue #1

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**